src/applications/edu-benefits/0994/components/HighTechEmploymentTypeView.jsx

```diff
@@ -0,0 +1,151 @@
+      false,
+    );
+    setHighTechEmploymentType(
+      formData,
+      highTechEmploymentTypeIds.computerSoftware.value,
+      false,
+    );
+    setHighTechEmploymentType(
+      formData,
+      highTechEmploymentTypeIds.informationSciences.value,
+      false,
+    );
+    setHighTechEmploymentType(
+      formData,
+      highTechEmploymentTypeIds.mediaApplication.value,
+      false,
+    );
+  } else {
+    setHighTechEmploymentType(
+      formData,
+      highTechEmploymentTypeIds.noneApply.value,
+      false,
+    );
+  }
+};
+
+class HighTechEmploymentTypeView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ...props.formData,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(e) {
+    const { id, checked } = e;
+    const formData = { ...this.props.formData };
+    handleNoneApply(id, formData);
+    setHighTechEmploymentType(formData, [id], checked);
+    this.props.onChange(formData);
+    this.setState(formData);
+  }
+
+  render() {
+    return (
+      <div>
+        <legend className="schemaform-label">
+          Which area best describes your high-tech work experience? (Check all
+          that apply.)
+        </legend>
+        <fieldset className="schemaform-field-template">
+          <TargetCheckboxWidget
+            id={highTechEmploymentTypeIds.computerProgramming.value}
+            value={this.props.formData.computerProgramming}
+            onChange={this.handleChange}
+            options={{
+              title: highTechEmploymentTypeIds.computerProgramming.title,
+            }}
+          />
+          <TargetCheckboxWidget
+            id={highTechEmploymentTypeIds.dataProcessing.value}
+            value={this.props.formData.dataProcessing}
+            onChange={this.handleChange}
+            options={{ title: highTechEmploymentTypeIds.dataProcessing.title }}
+          />
+          <TargetCheckboxWidget
+            id={highTechEmploymentTypeIds.computerSoftware.value}
+            value={this.props.formData.computerSoftware}
+            onChange={this.handleChange}
+            options={{
+              title: highTechEmploymentTypeIds.computerSoftware.title,
+            }}
+          />
+          <TargetCheckboxWidget
+            id={highTechEmploymentTypeIds.informationSciences.value}
+            value={this.props.formData.informationSciences}
+            onChange={this.handleChange}
+            options={{
+              title: highTechEmploymentTypeIds.informationSciences.title,
+            }}
+          />
+          <TargetCheckboxWidget
+            id={highTechEmploymentTypeIds.mediaApplication.value}
+            value={this.props.formData.mediaApplication}
+            onChange={this.handleChange}
+            options={{
+              title: highTechEmploymentTypeIds.mediaApplication.title,
+            }}
+          />
+          <TargetCheckboxWidget
```

Comment:
It might be useful to add some extra description text for this checkbox for screen readers, to let those users know that selecting this box will unselect the other boxes. Otherwise, they might not realize it, because the only feedback is visual.

cc @1Copenut